File: _posts/tutorials/2024-01-13-omeglin/snippets/main/java/io/javalin/omeglin/OmeglinMain.java
package io.javalin.omeglin;

import io.javalin.Javalin;
import io.javalin.http.staticfiles.Location;

public class OmeglinMain {
    public static void main(String[] args) {
        Javalin.create(config -> {
            config.staticFiles.add("src/main/resources/public", Location.EXTERNAL);
            config.router.mount(router -> {
                router.ws("/api/matchmaking", Matchmaking::websocket);
            });
        }).start(7070);
    }
}

File: _posts/tutorials/2024-01-13-omeglin/snippets/main/kotlin/io/javalin/omeglin/OmeglinMain.kt
package io.javalin.omeglin

import io.javalin.Javalin
import io.javalin.http.staticfiles.Location

fun main() {
    Javalin.create {
        it.staticFiles.add("src/main/resources/public", Location.EXTERNAL)
        it.router.mount{
            it.ws("/api/matchmaking", Matchmaker::websocket)
        }
    }.start(7070)
}
